index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Block-Chainference</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="style_d.css">
+  <script src="main.js" defer></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+
+<!--    Page for block chain coference 2023  -->
+
+<body>
+  <!--    bar menu and logo  -->
+  <header >
+    <section class="nav-bar-grey">
+      <a href="" class="nav-bar-grey-fabook"></a>
+      <a href="" class="nav-bar-grey-twitter"></a>
+      <a href="" class="nav-bar-grey-language">Language</a>
+      <a href="" class="nav-bar-grey-mypage">My Page</a>
+      <a href="" class="nav-bar-grey-logout">Logout</a>
+    </section>
+    <div class="nav-bar">
+      <section class="nav-menuContainer">
+        
+        <button class="bugerBtn">
+          <div class="burger-icon line1"></div>
+          <div class="burger-icon line2"></div>
+          <div class="burger-icon line3"></div>
+        </button>
+        <!--burger menu-->
+        <ul class="burger-menu">
+          <div class="burger-menu-style">
+              <li class="burger-menu-exit-button"><button class="exitButton"></button></li>
+              <li class="burger-menu-homePage-button"><a href="#home-page" class="burger-menu-home-page">Home Page</a></li>
+              <li class="burger-menu-aboutPage-button"><a href="#about-page" class="burger-menu-aboutPage-button">About</a></li>           
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">Join</a></li>
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">Sponsors</a></li>
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">News</a></li> 
+          </div>
+        </ul>
+        <!---->
+        <ul class="desktop-menu">
+          <li><a href="#home-page" class="desktop-menu-home">Home</a></li>
+          <li><a href="#about-page" class="desktop-menu-about">About</a></li>
+          <li><a href="" class="desktop-menu-about">Join</a></li>
+          <li><a href="" class="desktop-menu-about">Sponsors</a></li>
+          <li><a href="" class="desktop-menu-about">News</a></li> 
+        </ul>        
+      </section>
+      <div class="nav-logo-container">
+        <a href="#home-page" class="nav-logo-link">
+          <div class="nav-logo-img"></div>
+            <h1 class="nav-logo">Libra</h1>
+        </a>
```

Comment:
- [ ] Only one `h1` tag should be available in your codebase. 
- Using only one `h1` tag in a project limits accessibility, SEO optimization, and content organization. Multiple heading tags (such as `h2`, `h3`, etc.) should be used to create a hierarchical structure, improve search engine rankings, and enhance the user experience. The appropriate use of heading tags facilitates better comprehension, navigation, and clarity of the project's content.